kmip/services/server/session.py

```diff
@@ -51,6 +51,9 @@ def __init__(self, engine, connection, name=None):
             kwargs={}
         )
 
+        if name is None:
+            name = threading.current_thread().name
```

Comment:
Thank you very much for this developed answer, I'll be waiting for your _KmipServer_.